src/ui/mod.rs

```diff
@@ -847,14 +851,35 @@ fn draw_artist_albums<B>(f: &mut Frame<B>, app: &App, layout_chunk: Rect)
 where
     B: Backend,
 {
-    let current_route = app.get_current_route();
-    let highlight_state = (
-        current_route.active_block == ActiveBlock::Artist,
-        current_route.hovered_block == ActiveBlock::Artist,
-    );
+    let chunks = Layout::default()
+        .direction(Direction::Horizontal)
+        .constraints(
+            [
+                Constraint::Percentage(33),
+                Constraint::Percentage(33),
+                Constraint::Percentage(33),
+            ]
+            .as_ref(),
+        )
+        .split(layout_chunk);
 
-    if let Some(artist_albums) = &app.artist_albums {
-        let items = &artist_albums
+    if let Some(artist) = &app.artist {
+        let top_tracks = artist
+            .top_tracks
+            .iter()
+            .map(|artist| artist.name.to_owned())
+            .collect::<Vec<String>>();
+
+        draw_selectable_list(
+            f,
+            chunks[0],
+            &format!("<{}> - Top Tracks", &artist.artist_name),
```

Comment:
Would you be able to remove the angle brackets here `<>`?

```suggestion
            &format!("{} - Top Tracks", &artist.artist_name),
```

Reply:
I`ve deleted `<>`.